src/lib/annotations/AnnotationThread.js

```diff
@@ -612,6 +624,22 @@ class AnnotationThread extends EventEmitter {
     }
 
     /**
+     * Inserts user initials and updates point annotation icon with a
+     * color pseudo-unique to the user
+     *
+     * @private
+     * @param {Object} annotation - first annotation in the thread
+     * @return {void}
+     */
+    fillPointAnnotationIcon(annotation) {
+        // Update point annotation icon with user initials
+        this.element.innerHTML = annotatorUtil.getUserInitials(annotation.user.name);
```

Comment:
Can we not use `innerHTML` and instead create a span, and add the initials to it?
Or use `textContent` ?